### PR TITLE
Other Norfair Farms

### PR DESCRIPTION
**Pull request by blkerby**

### CODE REVIEW

File: region/norfair/east.json [hunk 1]
```diff
@@ -308,6 +308,25 @@
...
+                  "requires": [
+                    "h_heatProof",
+                    {"or": [
+                      "canWalljump",
+                      "SpaceJump",
+                      "h_canUseSpringBall"
+                    ]},
+                    {"refill": ["Energy", "Missile", "Super"]}
+                  ],
+                  "note": "You can stay on the left part of the room to avoid interacting with the Gerutas."
```
Comment (osse101) [inline region/norfair/east.json:326]: `Sm. Dessgeega`

File: region/norfair/east.json [hunk 2]
```diff
@@ -4298,6 +4451,34 @@
...
               ]
+            },
+            {
+              "id": 6,
+              "strats": [
+                {
+                  "name": "Alcoon and Multiviola Farm",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
```
Comment (osse101) [inline region/norfair/east.json:4462]: Screw, heatproof

File: region/lowernorfair/west.json [hunk 3]
```diff
@@ -2787,6 +2787,25 @@
...
+                  "requires": [
+                    "h_heatProof",
+                    "ScrewAttack",
+                    {"resetRoom": {
+                      "nodes": [1],
+                      "mustStayPut": false
+                    }},
+                    {"refill": ["Energy", "Super"]}
+                  ],
+                  "devNote": "Resetting using node 2 would also be possible, with Supers."
```
Comment (osse101) [inline region/lowernorfair/west.json:2805]: 👎

**Review by osse101 — COMMENTED** [review]
KiHunters with full beam would be ok to add.
Gold pirates with full beam+charge would work in this context.  They might get a shot off before you can charge a beam, but when resetting the room you are ready.

Curious how fast you get supers from bubble mountain wavers.  One per reset probably.

**Comment by blkerby** [comment]
Farming just the bottom 2 Wavers each with 24% Super drop rate, it would be an expected 0.5 Supers per cycle, so a little slow I guess. Should we keep it?

**Comment by blkerby** [comment]
Makes sense, removed the Waver farm from the bottom of the room now. Like you said, it's slow, and with canDelayedWalljump you can reach the Cacatac/Waver farm above anyway. I'm on the fence about the ripper: it's a bit far away down in the corner, and for low difficulty settings it may not be well known that it's a good source of supers/energy. On higher settings, again the Cacatac/Waver farm on top is reachable so I think we could probably say that covers it.

**Pull request merged**